unified_addrs.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+import sys; assert sys.version_info[0] >= 3, "Python 3 required."
+
+import math
+import struct
+
+from pyblake2 import blake2b
+from bech32m import bech32_encode, bech32_decode, convertbits, Encoding
+
+from tv_output import render_args, render_tv, Some
+from tv_rand import Rand
+from f4jumble import f4jumble, f4jumble_inv
+import sapling_key_components
+import orchard_key_components
+
+def encode_unified(receivers):
+    orchard_receiver = b""
+    if receivers['orchard']:
+        orchard_receiver = b"".join([b"\x03", len(receivers['orchard']).to_bytes(1, byteorder='little'), receivers['orchard']])
+
+    sapling_receiver = b""
+    if receivers['sapling']:
+        sapling_receiver = b"".join([b"\x02", len(receivers['sapling']).to_bytes(1, byteorder='little'), receivers['sapling']])
+
+    t_receiver = b""
+    if receivers['transparent']:
+        t_receiver = b"".join([b"\x01", len(receivers['transparent']).to_bytes(1, byteorder='little'), receivers['transparent']])
```

Comment:
This is only exercising P2SH. We should also generate test vectors with typecode `0x00` for P2PKH (which is likely to be more common). Since they are exclusive, and their values are the same length, we can just randomly pick one typecode or the other.